contracts/examples/adder/sc-config.toml

```diff
@@ -0,0 +1,4 @@
+[settings]
+proxy-paths = ["src/adder_proxy.rs"]
+
+[contracts.adder]
```

Comment:
Let's try to make this part not mandatory.

To explain: if this file is missing, a default config is created. Let's make this also work when this fie exists, but contracts are not configured.